src/common/QskNamespace.h

```diff
@@ -11,7 +11,11 @@
 
 namespace Qsk
 {
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+    Q_NAMESPACE_EXPORT(QSK_EXPORT)
```

Comment:
Hm, seems to convert "Q_DECL_EXPORT extern const QMetaObject staticMetaObject;" into "extern Q_DECL_EXPORT const QMetaObject staticMetaObject;" Is this worth the ifdefing ?

Or is there a specific tool, that makes use of redefining Q_NAMESPACE_EXPORT for some reason ?

Reply:
Of course the question is if it is worth to support Qt <= 5.14 any longer.

Reply:
I don't know the reason for the linker error yet but it is also a problem in Qt 5.15.2. Since the error hints `__declspec(dllimport)` in `squiekskin.lib` I double checked the compiler definitons for the `squiekskin` target and found that the following definitions are set: `/D "QSK_SQUIEK_MAKEDLL" /D "QSK_DLL"`

This seems correct in my scenario where qskinny shall be build as a shared library and imported in the skin therefore **not** setting  `QSK_MAKEDLL`.

For now lets treat this PR as a reminder / draft